### DISCUSSION TITLE
best.jpg not working on 0.11-beta4?

### DISCUSSION THREAD

**Discussion by sycophantic** [blakeblackshear/frigate#3364] [General]
I updated from 0.10 and now all my best.jpg images are 175x175 black images. Everything else seems to be working fine besides that. I don't see any one else saying anything about this, so I wonder if there is something odd about my conf?

**Comment by NickM-27** [comment]
They're no longer "best" and instead use the thumbnail from the most recent events of each label. Being black box likely indicates no event was found for that label on that camera

**Reply by sycophantic** [reply]
I guess thumbnail.jpg is no longer used either? The webui also shows blackboxes. There have been detections.

![image](https://user-images.githubusercontent.com/634076/174482214-acc12213-8f69-41c9-b863-91ae4c30a77d.png)

**Reply by NickM-27** [reply]
Well there's multiple thumbnail.jpg endpoints but none were removed

So the thumbnails on the events page don't show either?

Can you please paste your frigate logs and a screenshot of any browser logs (inspect element -> console)

**Comment by sycophantic** [comment]
The thumbnails on the events page works fine, this is on the cameras page for a camera.

The frigate logs don't so much for the webui. Just a websocket opening.

I'm going to rewrite my logic for the image grabbing, so this issue won't impact me anymore. I was more curious if I missed something while reading the changelogs.

[console-export-2022-6-19_9-17-10.txt](https://github.com/blakeblackshear/frigate/files/8935582/console-export-2022-6-19_9-17-10.txt)
